specs/apiPort.spec.js

```diff
@@ -47,4 +44,75 @@ describe('apiPort', () => {
       expect(() => apiPort.set('some', undefined, false)).not.to.throw()
     })
   })
+  describe('resolve', () => {
+    /* eslint-disable no-template-curly-in-string */
+    it('should handle tokens properly', () => {
+      const apiPOrt = new ApiPort()
+      apiPOrt.set('numToken', 555)
+      apiPOrt.set('boolToken', true)
+      apiPOrt.set('strToken', 'this is a token')
+
+      expect(apiPOrt.resolve('${numToken}')).to.be.equal(555)
+      expect(apiPOrt.resolve('${boolToken}')).to.be.equal(true)
+      expect(apiPOrt.resolve('${strToken}')).to.be.equal('this is a token')
+
+      expect(apiPOrt.resolve('Hello ${strToken} ${numToken} - ${boolToken}')).to.be.equal('Hello this is a token 555 - true')
```

Comment:
This line is too long but is not being flagged by eslint - how would I add a rule for that?